Documentation/api-reference.md

```diff
@@ -134,6 +134,9 @@ Identity.getUrlVariables(new AdobeCallback<String>() {
 
 Registers the Identity for Edge Network extension with the Mobile Core extension.
```

Comment:
can you include README.md under Documentation folder with table of contents? e.g. https://github.com/adobe/aepsdk-edge-android/tree/dev-v2.0.0/Documentation